composer.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "WP-API/WP-API",
+  "description":"JSON-based REST API for WordPress, developed as part of GSoC 2013.",
+  "keywords":["search"],
+  "authors":[
+	{
+	  "name":"Ryan McCue",
+	  "homepage":"http://ryanmccue.info/"
+	}
+  ],
+  "type": "wordpress-plugin",
+  "require": {
+    "composer/installers": "v1.0.6",
+    "php": ">=5.3.2"
```

Comment:
> I think the PHP requirement in composer.json should match the version of PHP that's actually required by WP-API.

I agree; however, where is the requirement actually checked? If it's checked at install-time, then it doesn't matter that it's >5.2.4, and IIRC it's not checked at run-time.

(Would still merge a PR to change it anyway. :) )